pom.xml

```diff
@@ -22,7 +17,7 @@
         <dependency>
             <groupId>org.junit.jupiter</groupId>
             <artifactId>junit-jupiter-engine</artifactId>
-            <version>5.3.1</version>
+            <version>5.6.0</version>
```

Comment:
Is it a part of this PR?

Reply:
Not necessarily. Side effect. Should I improve? delete?

Reply:
If the project works fine, it can stay :)

Reply:
It's work fine. I check it for two Maven version (current and (current - 1)) and for OpenJDK: 8, 11, 13.

Reply:
And I chekced it for the latest versions of Intellij. Because Maven-Maven and Maven-Intellij are two different things - unfortunately. I had problems with this for maven-failsafe-plugin and now I always check.